documentation/asciidoc/accessories/camera/camera_hardware.adoc

```diff
@@ -43,7 +43,8 @@ WARNING: Cameras are sensitive to static. Earth yourself prior to handling the P
 
 === Connect the Camera
 
-The flex cable inserts into the connector labelled CAMERA on the Raspberry Pi, which is located between the Ethernet and HDMI ports. The cable must be inserted with the silver contacts facing the HDMI port. To open the connector, pull the tabs on the top of the connector upwards, then towards the Ethernet port. The flex cable should be inserted firmly into the connector, with care taken not to bend the flex at too acute an angle. To close the connector, push the top part of the connector towards the HDMI port and down, while holding the flex cable in place.
+The flex cable inserts into the connector labelled CAMERA on the Raspberry Pi, which, on Raspberry Pi models 1-5, is located between the Ethernet and HDMI ports. The cable must be inserted with the metallised contacts facing the internal metallised contacts in the on-board connector. These internal contacts are on the opposite side to the movable tab/clamp/latch. The correct cable orientation can be achieved by ensuring that the plastic stiffener on the end of the cable is in contact with the tab/clamp/latch.  To open the connector, pull the tabs on the top of the connector upwards, then away from the fixed body of the connector. The flex cable should be inserted firmly and squarely into the connector, with care taken not to bend the flex at too acute an angle. To close the connector, push the top part of the connector evenly and firmly until it latches, while holding the flex cable in place.
```

Comment:
"labelled CAMERA on the Raspberry Pi" -> "labelled CAMERA on the Raspberry Pi (or labelled CAM/DISP on Raspberry Pi 5)" ?